pkg/storage/endpoint/tso_keyspace_group.go

```diff
@@ -80,6 +81,14 @@ type KeyspaceGroupMember struct {
 	Priority int    `json:"priority"`
 }
 
+// CompareAddress compares the address with the given address.
+// It compares the address without the scheme.
+// Otherwise, it will not work when we update the scheme from http to https.
+// Issue: https://github.com/tikv/pd/issues/8284
+func (m *KeyspaceGroupMember) CompareAddress(addr string) bool {
```

Comment:
Why not directly use EqualBaseURLs?

Reply:
Avoid to add comments anywhere, so wrap it.